content/en/building/reference/app-settings/token_login.md

```diff
@@ -11,17 +11,28 @@ aliases:
 ---
 
 Login via SMS settings are defined under the `token_login` key, as an object supporting the following properties:
+
+> **Note:**  
+> As of CHT 5.0.0, the `app_url` config is **required** when enabling token login. If `token_login.enabled` is `true` and `app_url` is not set, token login will not work and errors will be thrown.  
+> Example:
+> ```json
+> "app_url": "https://example.org"
+> ```
+
 ## `app_settings.json .token_login`
 | property         | type | required       | description                                                                                                                                                                              |
 |------------------|------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | enabled | Boolean | yes | Enables or disables token_login deployment-wide. When this is false, users can't be updated to use token_login and any requests to login with a token link will fail.  |
 | translation_key | String | yes | Translation key for the information (helper) sms message that the user receives, along with their token-login link |
+| app_url | String | yes | **Required as of 5.0.0 when token_login.enabled is true** The base URL of your CHT instance, used to generate login links. Example: "https://example.org" |
+* Only required if token_login.enabled is true.
```

Comment:
I think the callout above is sufficient.  (Plus the `app_url` is already documented over in https://docs.communityhealthtoolkit.org/building/reference/app-settings/#app_settingsjson)

```suggestion
```